src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/prison/Prisoner.kt

```diff
@@ -1,5 +1,8 @@
 package uk.gov.justice.digital.hmpps.hmppshdcapi.licences.prison
 
+import com.fasterxml.jackson.annotation.JsonFormat
+import java.time.LocalDateTime
```

Comment:
used `LocalDateTime` rather than `LocalDate` to be able to easily compare with `today` which is used to persist `deletedAt` with a `LocalDateTime` in the db

Reply:
this should be LocalDate as we only get passed back the date component.. you can do `LocalDate.now()` to get today.

(if you have a local date time you can always call `toLocalDate` on it)

Reply:
Cool, have made TUSED and LED `LocalDate`